src/patternfly/sass-utilities/mixins.scss

```diff
@@ -201,3 +201,18 @@
     #{$value}: #{$prop};
   }
 }
+
+// Animate tab focus removal
+@mixin pf-animate-remove-tab-focus($element, $delay: $pf-global--TransitionDuration) {
```

Comment:
I wonder if the name for this should be more generic? Like `pf-hide-delay` or something like that?